localflavor/ro/forms.py

```diff
@@ -3,26 +3,20 @@
 Romanian specific form helpers.
 """
 from __future__ import absolute_import, unicode_literals
-
 import datetime
 import re
-
 from django.core.validators import EMPTY_VALUES
 from django.forms import ValidationError, Field, RegexField, Select
 from django.utils.translation import ugettext_lazy as _
-
 from ..generic.forms import IBANFormField
 from .ro_counties import COUNTIES_CHOICES
 
-
```

Comment:
Please keep the spaces in the imports to match the rest of localflavor. A proper solution to getting consistent imports would be to setup something with isort for the whole project but there are other things that I think are more important right now. Thanks.